tldr/cross_file_calls/parsers/__init__.py

```
@@ -0,0 +1,104 @@
+"""
+Language-specific parsers for cross-file call analysis.
+"""
+
+from typing import Dict, List, Optional, Type
```

Ruff (0.14.11)

`typing.Dict` is deprecated, use `dict` instead

(UP035)